src/views/Profile/Avatar.tsx

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/react'
+import { PersonIcon } from '@radix-ui/react-icons'
+import { Flex } from '@radix-ui/themes'
+
+export function Avatar() {
```

Comment:
I know there's a `Avatar` component in radix-ui, but it only supports `solid` and `soft` variants which looked to dense IMO.